x/evm/types/statedb.go

```diff
@@ -243,6 +243,22 @@ func (csdb *CommitStateDB) SubRefund(gas uint64) {
 	csdb.refund -= gas
 }
 
+// AddAddressToAccessList is currently unimplemented
+func (csdb *CommitStateDB) AddAddressToAccessList(_ ethcmn.Address) {}
```

Comment:
I think we need to add this functionality since it is used for supporting [EIP2929](https://eips.ethereum.org/EIPS/eip-2929). I think there are additional journal entries too

Reply:
okay, I'll work on implementing it